SwissTransferFeatures/TransferDetailsView/EmptyStates/ExpiredTransferView.swift

```diff
@@ -16,18 +16,37 @@
  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+import STCore
 import STResources
 import SwiftUI
 import SwissTransferCoreUI
 
 struct ExpiredTransferView: View {
     @Environment(\.dismiss) private var dismiss
+    private let status: TransferStatus
+    private let transfer: TransferUi?
```

Comment:
I would prefer using an enum like `enum ExpirationType` and a case `downloadLimit(Int?)` do we don't have to give two parameters to the view